codechefcli/problems.py

```diff
@@ -306,6 +306,45 @@ def print_problem_tags(tags):
         print(SERVER_DOWN_MSG)
 
 
+def get_ratings(country, institution, institution_type, lines, page):
+    """
+    :desc: displays the ratings of users. Result can be filtered according to
+    the country, institution, institution_type and sets. `line` decide the
+    number of lines to be shown. `page` tells which page of the result is to be shown
+    :param: values of all filters, lines to be displayed, page no. to be displayed
+    """
+    session = get_session()
+    url = BASE_URL + '/api/ratings/all?sortBy=global_rank&order=asc&page='
+    url += str((page or 1)) + '&itemsPerPage=' + str((lines or 20))
+    if country or institution or institution_type:
+        url += '&filterBy='
+
+    if country:
+        url += 'Country=' + country + ";"
+
+    if institution:
+        institution = institution.title()
+        url += 'Institution=' + institution + ";"
+
+    if institution_type:
+        url += 'Institution type=' + institution_type + ";"
+
+    req_obj = request(session, 'GET', url)
+    data_rows = [RATING_HEADINGS]
+    if req_obj.status_code == 200:
+        ratings = req_obj.json().get('list')
```

Comment:
Will `list` key always be in the response `dict` ?

Reply:
i tried many combinations...it had 'list' key in each (was empty in some, but thats all)